spot_driver/src/spot_driver/spot_wrapper.py

```diff
@@ -674,6 +695,238 @@ def navigate_to(self, upload_path,
 
         return resp
 
+    # Arm ############################################
+    def check_arm_power_and_stand(self):
```

Comment:
```suggestion
    def ensure_arm_power_and_stand(self):
```

Does `self._robot.power_on` also cause spoto to stand up? If not, then I'd rename this function to `ensure_arm_and_robot_power` or something similar.

Reply:
```suggestion
    def check_arm_power_and_stand(self, stand_up = False):
           [CurrentCode]
            if not self._is_standing:
                self._logger.info("Commanding Spot to stand")
                robot_command.blocking_stand(command_client=self._robot_command_client,
                                             timeout_sec=10.0)
                self._logger.info("Spot standing")
            else:
                self._logger.info("Spot is already standing")
```

Reply:
Yes, this function makes spot stands as well. I just made the changes